integration/substrate/asserts.spec.ts

```diff
@@ -29,7 +29,7 @@ describe('Deploy asserts contract and test', () => {
         expect(res0.output?.toJSON()).toEqual(1);
 
         let res1 = await query(conn, alice, contract, "testAssertRpc");
-        expect(res1.result.toHuman()).toEqual({ "Err": { "Module": { "error": "0x0b000000", "index": "8" } } });
+        expect(res1.result.toHuman()).toEqual({ "Err": { "Module": { "error": "0x0b000000", "index": "7" } } });
```

Comment:
I pinned the contract node image recently, so this should not change out of nowhere?

Reply:
I was running tests on a different image locally, sorry for that

Reply:
No problem